site/gatsby-site/blog/incident-report-2022-october/index.fr.mdx

```diff
@@ -1,143 +1,142 @@
+Ces événements de ces dernières années mettent en évidence des thèmes communs de préoccupations en matière de vie privée qui concernent de plus en plus les systèmes juridiques garantissant des droits à la vie privée et à la protection des données.
 
-_– by Janet Schwartz_
+_- par Janet Schwartz_
 
-## 👇 Diving Deeper
+## 👇 Plonger plus profondément
 
-* All new incidents _added_ to the database in the last month, grouped by topic:
-    - **Privacy & surveillance**: [#354](/cite/354), [#357](/cite/357), [#360](/cite/360), [#361](/cite/361), [#371](/cite/371), [#372](/cite/372), [#377](/cite/377)
-    - **Facial recognition:** [#358](/cite/358), [#365](/cite/365), [#368](/cite/368), [#373](/cite/373), [#375](/cite/375), [#385](/cite/385)
-    - **Bias & discrimination:** [#355](/cite/355), [#356](/cite/356), [#359](/cite/359), [#367](/cite/367), [#374](/cite/374), [#383](/cite/383/)
-    - **Social media:** [#362](/cite/362), [#363](/cite/363), [#366](/cite/366), [#380](/cite/380), [#382](/cite/382)
-    - **Impactful errors:** [#364](/cite/364), [#379](/cite/379), [#384](/cite/384)
-    - **Unfair competition:** [#370](/cite/370), [#376](/cite/376)
-    - **Autonomous vehicles:** [#378](/cite/378), [#381](/cite/381)
-    - **Labor displacement:** [#369](/cite/369)
-* Explore clusters of similar incidents in [Spatial Visualization](/summaries/spatial)
-* Check out [Table View](/apps/incidents) for a complete view of all incidents
-* Learn about alleged developers, deployers, and harmed parties in [Entities Page](/entities)
+* Tous les nouveaux incidents _ajoutés_ à la base de données au cours du mois dernier, regroupés par sujet :
+    - **Confidentialité et surveillance** : [#354](/cite/354), [#357](/cite/357), [#360](/cite/360), [#361](/cite/ 361), [#371](/cite/371), [#372](/cite/372), [#377](/cite/377)
+    - **Reconnaissance faciale :** [#358](/cite/358), [#365](/cite/365), [#368](/cite/368), [#373](/cite/373 ), [#375](/cite/375), [#385](/cite/385)
+    - **Biais et discrimination :** [#355](/cite/355), [#356](/cite/356), [#359](/cite/359), [#367](/cite/ 367), [#374](/cite/374), [#383](/cite/383/)
+    - **Réseaux sociaux :** [#362](/cite/362), [#363](/cite/363), [#366](/cite/366), [#380](/cite/380 ), [#382](/cite/382)
+    - **Erreurs impactantes :** [#364](/cite/364), [#379](/cite/379), [#384](/cite/384)
+    - **Concurrence déloyale :** [#370](/cite/370), [#376](/cite/376)
+    - **Véhicules autonomes :** [#378](/cite/378), [#381](/cite/381)
+    - **Déplacement de main-d'œuvre :** [#369](/cite/369)
+* Explorez des groupes d'incidents similaires dans [Visualisation spatiale](/summaries/spatial)
+* Consultez [Table View](/apps/incidents) pour une vue complète de tous les incidents
+* En savoir plus sur les développeurs, les déployeurs et les parties lésées présumés dans [Entities Page](/entities)
```

Comment:
I think "Table View" and "Entities Page" should probably be translated. "Entities" is currently untranslated in the sidebar, so perhaps we should try to synchronize them in this PR.

![image](https://user-images.githubusercontent.com/25443411/202018701-468309ac-01fe-46b8-9f7c-9b6d646fc650.png)